autoload/neomake.vim

```diff
@@ -260,9 +271,10 @@ function! s:AddExprCallback(maker) abort
 
         " On the first valid error identified by a maker,
         " clear the existing signs in the buffer
-        call s:CleanSignsAndErrors()
+        call s:CleanSignsAndErrors(entry.bufnr)
```

Comment:
It seems like this means that a buffer can't have signs/errors for a `:Neomake` run and a `:Neomake!` run concurrently. Is that right?